### ISSUE TITLE
after 2.0.1 update error: 'ERR_UNHANDLED_REJECTION'

### ISSUE THREAD

**Issue by ngosang** [FlareSolverr/FlareSolverr#209]
### Discussed in https://github.com/FlareSolverr/FlareSolverr/discussions/208

<div type='discussions-op-text'>

<sup>Originally posted by **jprokos** October 25, 2021</sup>
Just removed repo and reinstalled and I am still getting this error.
macOS Catalina.
version 2.0.0 was working

```
10/24/21_20:08:59 /Users/john/GitHub/FlareSolverr git(master)
$ npm -v
8.1.0
10/24/21_20:11:00 /Users/john/GitHub/FlareSolverr git(master)
$ node -v
v17.0.1
```

```
> flaresolverr@2.0.1 start
> node ./dist/server.js

2021-10-24T20:08:47-05:00 INFO FlareSolverr v2.0.1
2021-10-24T20:08:47-05:00 INFO Testing web browser installation...
node:internal/process/promises:246
          triggerUncaughtException(err, true /* fromPromise */);
          ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "#<ErrorEvent>".] {
  code: 'ERR_UNHANDLED_REJECTION'
}

Node.js v17.0.1```
```
So I downgraded to version 2.0.0 and still have the same error:
```
10/24/21_21:54:14 /Users/john/GitHub/FlareSolverr git(tags/v2.0.0)
$ npm outdated
Package            Current    Wanted   Latest  Location                       Depended by
@types/node       14.17.27  14.17.29  16.11.4  node_modules/@types/node       FlareSolverr
@types/puppeteer     3.0.6     3.0.6    5.4.4  node_modules/@types/puppeteer  FlareSolverr
nodemon             2.0.13    2.0.14   2.0.14  node_modules/nodemon           FlareSolverr
puppeteer            3.3.0     3.3.0   10.4.0  node_modules/puppeteer         FlareSolverr
ts-node             10.3.0    10.4.0   10.4.0  node_modules/ts-node           FlareSolverr

10/24/21_21:59:31 /Users/john/GitHub/FlareSolverr git(tags/v2.0.0)
$ npm update
npm WARN old lockfile 
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
npm WARN old lockfile 
npm WARN old lockfile This is a one-time fix-up, please be patient...
npm WARN old lockfile 

added 46 packages, removed 104 packages, changed 70 packages, and audited 650 packages in 14s

51 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

10/24/21_22:00:29 /Users/john/GitHub/FlareSolverr git(tags/v2.0.0)
$ npm outdated
Package            Current    Wanted   Latest  Location                       Depended by
@types/node       14.17.29  14.17.29  16.11.4  node_modules/@types/node       FlareSolverr
@types/puppeteer     3.0.6     3.0.6    5.4.4  node_modules/@types/puppeteer  FlareSolverr
puppeteer            3.3.0     3.3.0   10.4.0  node_modules/puppeteer         FlareSolverr

10/24/21_22:00:39 /Users/john/GitHub/FlareSolverr git(tags/v2.0.0)
$ npm start

> flaresolverr@2.0.0 start
> node ./dist/server.js

2021-10-24T22:01:49-05:00 INFO FlareSolverr v2.0.0
2021-10-24T22:01:49-05:00 INFO Testing web browser installation...
node:internal/process/promises:246
          triggerUncaughtException(err, true /* fromPromise */);
          ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "#<ErrorEvent>".] {
  code: 'ERR_UNHANDLED_REJECTION'
}

Node.js v17.0.1
```</div>

**Comment by ngosang** [comment]
I just released v2.0.2
It's not going to resolve your issue but it will give you a clear error message.

**Comment by jprokos** [comment]
Actually it's working...
If anything changes I will open a new issue.

```
10/31/21_17:01:26 /Users/john
$ cd /Users/john/GitHub/FlareSolverr 

10/31/21_17:01:29 /Users/john/GitHub/FlareSolverr git(master)
$ export PUPPETEER_PRODUCT=firefox

10/31/21_17:01:43 /Users/john/GitHub/FlareSolverr git(master)
$ npm install

added 707 packages, and audited 708 packages in 55s

51 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

10/31/21_17:02:43 /Users/john/GitHub/FlareSolverr git(master)
$ node node_modules/puppeteer/install.js
Requesting latest Firefox Nightly version from https://archive.mozilla.org/pub/firefox/nightly/latest-mozilla-central
Firefox Nightly is already in /Users/john/GitHub/FlareSolverr/node_modules/puppeteer/.local-firefox/mac-95.0a1; skipping download.

10/31/21_17:02:51 /Users/john/GitHub/FlareSolverr git(master)
$ npm run build

> flaresolverr@2.0.2 build
> tsc


10/31/21_17:03:04 /Users/john/GitHub/FlareSolverr git(master)
$ npm start

> flaresolverr@2.0.2 start
> node ./dist/server.js

2021-10-31T17:03:11-05:00 INFO FlareSolverr v2.0.2
2021-10-31T17:03:11-05:00 INFO Testing web browser installation...
2021-10-31T17:03:18-05:00 INFO FlareSolverr User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:95.0) Gecko/20100101 Firefox/95.0
2021-10-31T17:03:18-05:00 INFO Test successful
2021-10-31T17:03:18-05:00 INFO Listening on http://0.0.0.0:8191
```
